src/components/questions/AddQuestionForm.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './questions.module.css';
+
+const QuestionForm = (props) => {
+  if (props.show) {
+    return (
+      <div className={styles.modal}>
+        <div className={styles.content}>
+          <form className={styles.questionForm}>
+            <label> Your Question: </label>
+            <input type='text' name='question' maxlength='1000'></input>
+            <label> Your Nickname: </label>
+            <input type='text' placeholder='Example: jackson11!' maxlength='60'></input>
+            <span className={styles.inputSubtext}> For privacy reasons, do not use your full name or email address </span>
+
+            <label> Your Email: </label>
+             <input type='text' placeholder='Why did you like the product or not?' maxlength='60'></input>
+            <span className={styles.inputSubtext}> “For authentication reasons, you will not be emailed”</span>
+            <input type='submit' name='Submit Question'></input>
+            <div className={styles.actions}>
+              <button onClick={props.modalButtonClick}> Close </button>
+            </div>
+          </form>
+        </div>
+    </div>
+    )
+  } else {
+    return null;
```

Comment:
I didn't realize you could just return null. I like that and feel like it is probably a little cleaner than what I've been doing (Returning <></>

Reply:
Nice!